homeassistant/components/cast/media_player.py

```diff
@@ -103,13 +108,15 @@ def _fill_out_missing_chromecast_info(info: ChromecastInfo) -> ChromecastInfo:
     # Fill out missing information via HTTP dial.
     from pychromecast import dial
 
-    http_device_status = dial.get_device_status(info.host)
+    http_device_status = dial.get_device_status(
```

Comment:
I wonder if we should get rid of this, because it will never work for groups? 

The DIAL protocol only fetches the "main" cast device. If it's a group, and thus a virtual cast device, the DIAL info is off?

Reply:
OK, I'll leave that for a separate cleanup PR though?

Reply:
Sounds good